### DISCUSSION TITLE
Remove the "For You" tab from the feed

### DISCUSSION THREAD

**Discussion by dgbauleo** [community/community#54486] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I don't know who had this idea, but it's terrible and I'm actually surprised it was even considered, let alone implemented and pushed to production.
Please give an option for us who want to focus on work to completely disable it. GitHub is not a social network, I don't want to get updates from people I worked with for one week in a small project 2 years ago.

**Comment by AliKhadivi** [comment]
I am concerned about the privacy implications of the "For You" tab. According to the [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement), GitHub collects information directly from me for my registration, payment, transactions, and user profile. It also automatically collects from me my usage information, cookies, and device information. GitHub may also collect personal data from third parties. I do not want GitHub to track my activities and the projects that I am looking at without my permission. I want to have more control over what data GitHub collects from me and how it uses it. I also want to know how GitHub protects my personal data from unauthorized access or disclosure. Please respect my privacy and give me the option to opt out of the "for you" tab and any other nonessential cookies or tracking technologies. Thank you.
